packages/ui/package.json

```
@@ -10,10 +10,17 @@
         "build": "d2-app-scripts build"
     },
     "dependencies": {
-        "@dhis2/ui-constants": "5.3.0",
-        "@dhis2/ui-icons": "5.3.0",
-        "@dhis2/ui-core": "5.3.0",
-        "@dhis2/ui-forms": "5.3.0",
-        "@dhis2/ui-widgets": "5.3.0"
+        "@dhis2/ui-constants": "^5.3",
+        "@dhis2/ui-icons": "^5.3",
+        "@dhis2/ui-core": "^5.3",
+        "@dhis2/ui-forms": "^5.3",
+        "@dhis2/ui-widgets": "^5.3"
+    },
+    "peerDependencies": {
+        "react": "^16.8",
+        "react-dom": "^16.8",
+        "styled-jsx": "^3.2",
+        "@dhis2/app-runtime": "^2.2.1",
```

would it change anything if we made this as following?
```suggestion
        "@dhis2/app-runtime": "^2.2",
```

---

It shouldn't matter too much, since `2.2.1` is compatible with `2.0.0`, but yeah we should be consistent and use `^2.2.1` to be safe, I think.